cmd/cli/cmd/alpha/content/interface.go

```diff
@@ -0,0 +1,42 @@
+package content
+
+import (
+	"log"
+
+	"capact.io/capact/internal/cli"
+	"capact.io/capact/internal/cli/alpha/content"
+	"capact.io/capact/internal/cli/heredoc"
+	"github.com/spf13/cobra"
+)
+
+var interfaceCfg content.InterfaceConfig
+
+// NewInterface returns a cobra.Command to bootstrap new Interface manifests.
+func NewInterface() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "interface [PREFIX] [NAME]",
```

Comment:
I think splitting prefix and name could be confusing for users. Maybe the user should provide a full path, even including `cap.interface.`?

E.g.:

```
capact alpha manifest-gen interface cap.interface.database.postgresql.install
```